engine/gamesys/src/dmsdk/gamesys/render_constants.h

```diff
@@ -17,6 +17,7 @@
 
 #include <stdint.h>
 #include <dmsdk/dlib/hash.h>
+#include <dmsdk/gameobject/gameobject.h>
```

Comment:
What is this needed for?

Reply:
It's needed for dmGameObject::PropertyResult, PropertyDesc and PropertyVar